Back-end/Service/GenreService.cs

```diff
@@ -0,0 +1,120 @@
+﻿using DENMAP_SERVER.Entity;
+using DENMAP_SERVER.Repository;
+using MySql.Data.MySqlClient;
+
+namespace DENMAP_SERVER.Service
+{
+    internal class GenreService
+    {
+        string connectionString = "Server=34.116.253.154;Port=3306;Database=chat_database;Uid=app_user;Pwd=&X9fT#7vYqZ$4LpR;";
+
+        private GenreRepository genreRepository = new GenreRepository();
+
+        public int AddGenre(string name)
+        {
+            int id = 0;
+
+            using (MySqlConnection connection = new MySqlConnection(connectionString))
+            {
+                connection.Open();
+
+                try
+                {
+                    id = genreRepository.addGenre(connection, name);
+                }
+                catch (Exception ex)
+                {
+                    throw new Exception("Database error.\nError:" + ex.Message);
```

Comment:
Won't that show MySQL database error message to users? 
If so, please log such messages, but don't show them to the user

Reply:
at this stage it was useful for debugging, but in general yes, just logs are enough